hs_core/templates/pages/genericresource.html

```diff
@@ -828,13 +831,34 @@ <h3>Resource Specific</h3>
       const SUPPORTE_FILE_TYPES = "{{ cm.get_supported_upload_file_types }}";
       const ALLLOW_MULTIPLE_FILE_UPLOAD = "{{ cm.allow_multiple_file_upload }}";
       const RESOURCE_MODE = "{% if resource_edit_mode %}Edit{% else %}View{% endif %}";
-      const CURRENT_USER_ID = {{ request.user.id }};
+      const CURRENT_USER_ID = {{ request.user.id|default:"null" }};
```

Comment:
I really think these blocks should be set from an ajax call rather than the intial page template.  It should also speed up our initial response quite a bit.